utopian-io/v2.utopian.io
# Feature/category/ut 417

### Verify that

### **Test plan**

_Get the categories in English_

* Call http://localhost:5000/v1/categories/en

**Expected result:** The API return 12 categories with the text in English

_Get the categories in French_

* Call http://localhost:5000/v1/categories/fr

**Expected result:** The API return 12 categories with the text in French

_Get the categories in German. Oh no! translations are missing! English fallback then_

* Call http://localhost:5000/v1/categories/de

**Expected result:** The API return 12 categories with the text in English

_Get the subcategories of development in English_

* http://localhost:5000/v1/subcategories/en/development

**Expected result:** The API return 1 subcategory with the text in English

_Get the subcategories of development in French_

* http://localhost:5000/v1/subcategories/fr/development

**Expected result:** The API return 1 subcategory with the text in French

_Get the subcategories of development in German, English fallback_

* http://localhost:5000/v1/subcategories/de/development

**Expected result:** The API return 1 subcategory with the text in English
